docs/essentials/overriding.md

```diff
@@ -0,0 +1,151 @@
+# Overriding
+
+Overriding allows for the easy replacement of parts of SFX, thereby modifying, adding, or removing functionality. For example, you can easily override how adding to cart works, how header/footer looks, how the catalog is displayed etc.
+
+Overriding works in a file-based manner, so basically any file (page, component, service, composable, etc.) can be overridden.
+
+::: tip
+Overriding rules are managed in the `storefront-x.config.js` file based on the position of imported module. The file from a later module will be used instead of the file from an earlier module. If only one module is imported, the file from that module will be used.
+:::
+
+Overriding is one of the primary feature that allows Storefront X to be extremely flexible and performant at the same time.
+
+## How to override?
+
+Imagine we have a default `module-a` with the `Header.vue` component in the `/organisms` concept.
```

Comment:
Dont complicate it with atomic design and concepts. Just say Header.vue is in the components directory. It doesnt have to be 1:1 as our demo.